src/test/java/com/bearAndPupperCo/sangenWrestlingApp/Repository/WrestlerRepoImplTest.java

```diff
@@ -0,0 +1,312 @@
+package com.bearAndPupperCo.sangenWrestlingApp.Repository;
+
+import com.bearAndPupperCo.sangenWrestlingApp.DTO.WrestlerMainTableDTO;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+public class WrestlerRepoImplTest {
+
+    @Autowired
+    private WrestlerRepoImpl wrestlerRepo;
+
+    private int page = 1;
+    private int size = 10;
+    private Integer brandId = 1;
+    private Integer lockerId = 1;
+    private String orderBy = "in_ring_name";
+    private String orderDirection = "ASC";
+
+    @Test
+    public void findWrestlerListByRingNameASC() {
```

Comment:
Simplify methods according to the variable being changed. 
p.ex you can combine findWrestlerListByRingNameASC and DESC, etc 
keep the complexity you find necessary for it to be readable and expandable

Reply:
Done